sunpy/util/decorators.py

```diff
@@ -182,3 +183,28 @@ def deprecate(obj, message=message, name=name, alternative=alternative):
         return deprecate(message)
 
     return deprecate
+
+
+class add_common_docstring(object):
+    """
+    A function decorator that will append and/or prepend an addendum
+    to the docstring of the target function.
+
+    If a dictionary(d) is provided as parameter then format by .format().
+    """
+    def __init__(self, append='', prepend='', d={}):
```

Comment:
Given that you check for the truth value of `append` and `prepend` in the `__call__` method, I think you would be better off defaulting these to `None` over `''`